atlasreader/data/README.md

```diff
@@ -0,0 +1,337 @@
+# Template Information
+
+## MNI152 T1 1mm Template
+
+The MNI152 T1 template is taken from FSL 5.0. It's kindly supplied by Andrew
+Janke. This template is derived from 152 structural images, averaged together
+after high-dimensional nonlinear registration into the common MNI152 co-ordinate
+system. It corresponds to the "[152 nonlinear 6th generation](http://www.bic.mni.mcgill.ca/ServicesAtlases/HomePage)" atlas.
+
+### Creation of template
+
+This atlas is a direct copy of the file `MNI152_T1_1mm_brain.nii.gz` packaged
+with FSL 5.0.
+
+### References
+
+The references are related to FSL 5.0:
```

Comment:
I’m confused by all these references. These are all the refs to cite each of the atlases shipped with FSL, right? So would we ask an `atlasreader` user to also cite all of them? Could we add a section (maybe under `License` for this section) that says how you would cite this?

I think we’re all wrestling with giving as much credit as we can while also not over burdening the user to reuse files that (if we’re being honest) aren’t cited in most use cases! I’m really glad we want to do better than the status quo, but citing all of the references below just when you use MNI152 doesn’t make sense to me.

Reply:
Yes, you're right. It might be a bit of an overkill. And I agree, the best approach is probably to have an additional section (at the end of the readme?) that specifies per atlas the references that should be mentioned (perhaps up to 3 max?).
As you also mention, we might want to make sure to only list the references connected to the atlases and not the whole software (FSL, SPM etc.)

Reply:
@KirstieJane - I moved the references to the end of the readme with https://github.com/miykael/atlasreader/pull/67/commits/51af88f9b394557c3f64b0d299672da1534018b4. Like this the citation of the atlases might be easier to access. We should nonetheless link in the main README.md to this file and explain that users have to cite the corresponding references and consider the licenses, no?